Misc/NEWS.d/next/Library/2024-11-18-19-03-46.gh-issue-126947.NiDYUe.rst

```diff
@@ -0,0 +1,2 @@
+Raise :exc:`TypeError` in :meth:`!_pydatetime.timedelta.__new__` arguments are not :class:`int` or :class:`float`, so that python
+implementation is in line with C implementation.
```

Comment:
Grammar fixes:
```suggestion
Raise :exc:`TypeError` in :meth:`!_pydatetime.timedelta.__new__` if the passed arguments are not :class:`int` or :class:`float`, so that the Python
implementation is in line with the C implementation.
```